plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -159,17 +158,17 @@ def get_timestamp_seconds(row):
     '''
     return row.get(Constants.TIMESTAMP) / 1000.0 / 1000.0
 
-class IsolationMgr:
+class IsolationMgr:#pylint: disable=too-many-instance-attributes,too-many-public-methods
```

Comment:
Please put the disable=XXX in the line prior to the error because it is less visible at the end of the line.